src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -231,16 +257,22 @@ private function fixDocBlock($content)
                         continue;
                     }
 
-                    $extraIndent = 2;
-
-                    if (in_array($currTag, self::$tagsWithName, true) || in_array($currTag, self::$tagsWithMethodSignature, true)) {
-                        $extraIndent = 3;
+                    $indent = $this->descriptionExtraIndent;
+                    if ('hint' === $this->descriptionAlign) {
+                        $indent += $tagMax + 2;
+                    } elseif ('name' === $this->descriptionAlign) {
+                        $indent += $tagMax + $hintMax + 3;
+                    } elseif ('description' === $this->descriptionAlign) {
+                        $indent += $tagMax + $hintMax + 3;
+                        if ($varMax) {
+                            $indent += $varMax + 1;
+                        }
                     }
 
                     $line =
                         $item['indent']
-                        .' *  '
-                        .str_repeat(' ', $tagMax + $hintMax + $varMax + $extraIndent)
+                        .' * '
+                        .str_repeat(' ', max(0, $indent))
```

Comment:
`max()` should be applied to `$this->descriptionExtraIndent` a few lines above as it is the only variable that can be negative:
```php
$indent = max(0, $this->descriptionExtraIndent);
```
This will prevent weird cases like e.g.:
```
$this->descriptionExtraIndent (-4) + $tagMax (6) + 2 = 4 instead of 8
```

Maybe even better would be to validate the `description_extra_indent` option instead.

Please also add a test case that covers this.

Reply:
Actually a negative extra indent makes no sense to me as well, but do we need to forbid it?